### PR TITLE
Update generic error assets

### PR DESCRIPTION
**Pull request by qroll**
**Changes**

- delete branch

**Changelog entry**

- Updated illustration for unsupported browser and 500 error in `ErrorDisplay`

**Additional information**

- You may refer to this [MOL-12262](https://jira.ship.gov.sg/browse/MOL-12262)
- Any other information you would like to include

### CODE REVIEW

**Comment by weili-govtech** [comment]
I notice the error 500 image is still using 416px width on desktop, I think there is a need to update the styled component too: https://github.com/LifeSG/react-design-system/blob/generic-error-assets/src/error-display/error-display.style.tsx#L17

**Comment by weili-govtech** [comment]
also notice the figma mock is using #484848 for the title colour, is there an intention to use it eventually?

**Comment by qroll** [comment]
> also notice the figma mock is using #484848 for the title colour, is there an intention to use it eventually?

good catch, it will follow the DS as #282828

File: src/error-display/error-display.style.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const Container = styled.div`
 
 export const Img = styled.img`
     position: relative;
-    width: 26rem;
+    width: 25rem;
```
Comment (keithtxw) [inline src/error-display/error-display.style.tsx:17]: You required confirmation on this yes?

Comment (qroll) [reply src/error-display/error-display.style.tsx:17]: yes

**Comment by qroll** [comment]
change to 400px/25rem is confirmed, mobile will remain as is

**Review by weili-govtech — APPROVED** [review]

**Pull request merged**